.github/actions/failover-test/action.yml

```
@@ -0,0 +1,28 @@
+name: Failover test
+description: Runs helm tests for the provided scenario
+inputs:
+  west:
+    description: podinfo west available
+    required: true
+  central:
+    description: podinfo central available
+    required: true
+  east:
+    description: podinfo east available
+    required: true
+runs:
+  using: composite
+  steps:
+  - shell: bash
+    run: |
+      helm upgrade -i -n linkerd-failover-tests --create-namespace --wait \
```

Super minor/tioli, but might be a little clearer to use the full flag name

```suggestion
      helm upgrade --install -n linkerd-failover-tests --create-namespace --wait \
```

---

:+1: